src/redux/books/books.js

```diff
@@ -1,5 +1,6 @@
 /* eslint-disable no-case-declarations */
```

Comment:
- [ ] Let's not disable linter rules. these are here to help us write clean code.

Reply:
Thanks for the code review 😃 The changes have been made 🏳️‍🌈